driver-testsuite/tests/Form/SelectTest.php

```diff
@@ -102,6 +102,43 @@ public function testSetValueMultiSelect()
     }
 
     /**
+     * Set values by Driver
+     *
+     * @dataProvider setValueByOptionTextProvider
+     */
+    public function testSetValueByOptionText($xpath, $value, $expectedResult)
+    {
+        $session = $this->getSession();
+        $session->visit($this->pathTo('/multiselect_form.html'));
+        $page = $session->getPage();
+        $select = $page->find('xpath', $xpath);
+
+        $select->setValue($value);
+        $this->assertEquals($expectedResult, $select->getValue());
+    }
+
+    public function setValueByOptionTextProvider()
+    {
+        return array(
+            'single select' => array(
+                'xpath'          => '//select[@name="select_number"]',
```

Comment:
1. Indentation in arrays isn't needed.
2. Key names aren't used by PHPUnit and can be removed.